contents/naturalLanguage/nl.md

```diff
@@ -0,0 +1,195 @@
+<frontmatter>
+  title: Introduction to Natural Language Processing (NLP)
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Natural Language Processing (NLP)
+Authors: [Lum Ka Fai Jeffry](https://github.com/j-lum)
+
+* [What is NLP?](#what-is-nlp)
+* [Core problems in NLP](#core-problems-in-nlp)
+   * [Syntax](#syntax)
+   * [Semantics](#semantics)
+   * [Relations](#relations)
+* [Applications of NLP](#applications-of-nlp)
+* [What's next](#whats-next)
+* [References](#references)
+
+
+## What is NLP?
+Natural Language Processing (NLP) is the set of methods for making human language accessible to computers.
+Recent advances in NLP have given rise to useful tools that have become embedded in our daily lives: 
+spam and phishing classification keeps inboxes sane<sup>[1](#footnote1)</sup>;
+automated chatbots lighten the load on customer support staff and provide customers with immediate feedback<sup>[2](#footnote2)</sup>;
+machine translation bridge the gap between cultures<sup>[3](#footnote3)</sup>. 
+ 
+NLP draws from many other fields of science, from formal linguistics to statistics. 
+The goal of NLP is to provide new computational capabilities around human language: for example, holding a conversation, summarizing an article, and so on.
+
+Even though the study of NLP covers a diverse range of tasks, most of them can be generalized to three themes: syntax, semantics and relations.
+
+## Themes in NLP
+
+### Syntax
+In order to leverage the power of modern machine learning techniques, raw text must be converted to a general-purpose linguistic structure.
+In English, words can be decomposed to _morphemes_, the minimal unit of meaning (e.g. unhappiness to three morphemes un-happy-ness)<sup>[4](#footnote4)</sup>.
+Similarly, sentences can be tagged with word-level _parts-of-speech_ which describe the meaning of each word in the context of that sentence.
+
+<p align="center">
+    <pic src="partsOfSpeech.png" alt="partsOfSpeech" width="90%">
+      <sub><i>Figure 1. Parts of Speech</i></sub>
+    </pic>
+</p>
+For example, 'word-level' is tagged as an adjective (JJ) and 'sentences' as a plural noun (NNS)<sup>[5](#footnote5)</sup>.
```

Comment:
in the preview, the footnote 5 seems to not work
<img width="197" alt="Screen Shot 2020-03-27 at 12 44 54 PM" src="https://user-images.githubusercontent.com/32864116/77722458-c38caa80-7028-11ea-8049-e16e8fa3d962.png">